### ISSUE TITLE
Consistently name context variable

### ISSUE THREAD

**Issue by DimitrisJim** [cosmos/ibc-go#7702]
Ref https://github.com/cosmos/ibc-go/pull/7261#discussion_r1888294760

> minor nit for naming these context vars with consistency ctx/goCtx

Would be nice to use same names across the board to avoid confusion.

Should be done after https://github.com/cosmos/ibc-go/pull/7261 and relevant PRs for it are merged.

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged/assigned
- [ ] Estimate provided

**Comment by iIvaki** [comment]
@DimitrisJim can I do this?

**Comment by technicallyty** [comment]
can we remove this from the olympus epic? trying to slim down the epic to whats strictly necessary

**Comment by gjermundgaraba** [comment]
Yeah, good point! Removing it from the epic and marking as nice-to-have

**Comment by hungdinh82** [comment]
@gjermundgaraba does this issue still need to be resolved?

**Comment by gjermundgaraba** [comment]
This issue was mistakenly reopened during some integration work, so closing again now.
